package.json

```diff
@@ -126,8 +128,5 @@
     "posttest": "npm run postpublish",
     "get-filename": "node scripts/get-filename.js",
     "add-icon-data": "node scripts/add-icon-data.js"
-  },
-  "engines": {
-    "node": ">=0.12.18"
```

Comment:
Why this has been removed?

Reply:
You can add it back and see the error from `n/no-unsupported-features/es-syntax`.

I think this is unnecessary since `0.12.18` is a really old version. We can totally ignore it.

Reply:
🤔 It seems there is something we have missed so far. There are multiple ways to use the npm package: the SDK with its TS types, the library's TS types, and the library itself, and ideally we should be able to set values of this field for each.

If is not possible, I think that we should ignore the rule to maintain proper compatibility with all supported versions, just for legacy reasons. Seems to me that do not hurts.

Reply:
Targeting the engine to the `0.12.18` is not accurate anymore. You can see tons of errors like:

![](https://github.com/simple-icons/simple-icons/assets/8186898/7b596a57-2234-4a02-9a8f-efd92af48367)

![](https://github.com/simple-icons/simple-icons/assets/8186898/95527ab3-13cf-46d5-8016-58d4c3ff4f6a)

Reply:
Could you set the rule to run only for distributed files?

Reply:
I've set the rule with `overrides`.

Reply:
I think the engine field can be removed because the `0.12.18` is not correct anymore.

Reply:
Most of the people is using the library, not the SDK. What happens if you readd `engines.node`? Seems to me that doesn't hurts.

Reply:
I would remove the incorrect field. If you really want this, we should use `>=12.16.0` instead.

We're using conditional exports in `packcage.json`. See https://nodejs.org/api/packages.html#conditional-exports.

Reply:
See this comment: https://github.com/simple-icons/simple-icons/pull/10643#discussion_r1530223931 We can check the Node.js version in the SDK itself because almost nobody is using it.